src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+body {
+  margin: 0;
+  min-width: 1024px;
+  max-width: 1200px;
```

Comment:
Why do you need this? Don't limit your header
![image](https://user-images.githubusercontent.com/96409563/197150218-2bf71a50-36e2-4687-9238-1b8f395b34f4.png)